ts/features/bonus/cgn/screens/CgnOtpScreen.tsx

```diff
@@ -0,0 +1,125 @@
+import { Dispatch } from "../../../../store/actions/types";
+import { cgnOtpDataSelector } from "../store/reducers/otp";
+import { cgnGenerateOtp } from "../store/actions/otp";
+import { useActionOnFocus } from "../../../../utils/hooks/useOnFocus";
+import LoadingSpinnerOverlay from "../../../../components/LoadingSpinnerOverlay";
+import { isLoading, isReady, RemoteValue } from "../../bpd/model/RemoteValue";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../components/core/typography/H1";
+import { OtpCodeComponent } from "../components/otp/OtpCodeComponent";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import { cancelButtonProps } from "../../bonusVacanze/components/buttons/ButtonConfigurations";
+import I18n from "../../../../i18n";
+import { clipboardSetStringWithFeedback } from "../../../../utils/clipboard";
+import { Body } from "../../../../components/core/typography/Body";
+import { Link } from "../../../../components/core/typography/Link";
+import { H4 } from "../../../../components/core/typography/H4";
+import { Otp } from "../../../../../definitions/cgn/Otp";
+import { NetworkError } from "../../../../utils/errors";
+import useOtpNotWorkingBottomSheet from "../components/otp/OtpNotWorking";
+import { navigateToCgnMerchantsList } from "../navigation/actions";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const getReadableTTL = (
+  otp: RemoteValue<Otp, NetworkError>
+): string | undefined => {
+  if (isReady(otp)) {
+    return otp.value.ttl > 60
+      ? I18n.t("bonus.cgn.otp.code.minutes", {
+          defaultValue: I18n.t("bonus.cgn.otp.code.minutes.other", {
+            minutes: Math.round(otp.value.ttl / 60)
+          }),
+          count: Math.round(otp.value.ttl / 60),
+          minutes: Math.round(otp.value.ttl / 60)
+        })
+      : I18n.t("bonus.cgn.otp.code.seconds", {
+          defaultValue: I18n.t("bonus.cgn.otp.code.seconds.other", {
+            seconds: otp.value.ttl
+          }),
+          count: otp.value.ttl,
+          seconds: otp.value.ttl
+        });
+  }
+  return undefined;
+};
+
+const CgnOtpScreen: React.FunctionComponent<Props> = (props: Props) => {
+  useActionOnFocus(props.generateOtp);
+
+  const readableTTL = getReadableTTL(props.otp);
+
+  const { present } = useOtpNotWorkingBottomSheet(props.navigateToMerchants);
+
+  return (
+    <LoadingSpinnerOverlay isLoading={isLoading(props.otp)}>
+      <BaseScreenComponent goBack contextualHelp={emptyContextualHelp}>
+        <SafeAreaView style={IOStyles.flex}>
+          <ScrollView style={[IOStyles.flex]} bounces={false}>
+            <View style={IOStyles.horizontalContentPadding}>
+              <H1>{"Usa codice sconto"}</H1>
```

Comment:
```suggestion
              <H1>{"Usa codice sconto"}</H1>
```
Use translation

Reply:
I created the entry in the locales files but missed to replace it!

adding with the rest of fixes